css/style.css

```diff
@@ -261,6 +251,68 @@ header h2 {
   align-items: center;
 }
 
+.mobile-menu-hide {
+  display: none;
+}
+
+.mobile-menu-show {
```

Comment:
- [ ] The menu has a blurred background, kindly add some opacity and blur to the background so it can be as close to the Figma design as possible

- Your design:
![Screenshot_20220510_221156](https://user-images.githubusercontent.com/25227022/167722954-f8577086-2a6e-4bd3-803d-5331caeb220f.png)

- Figma design:
![Screenshot_20220510_221131](https://user-images.githubusercontent.com/25227022/167722992-4904f898-444b-491c-afc5-9af9c69c77a3.png)